monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md

```diff
@@ -0,0 +1,129 @@
+# MONERO
+
+Monero este o valută digitală securizată,
+confidențială si nedetectabilă care este
+disponibilă tuturor.
+
+# SUMAR
+
+Utilizatorii pot cheltui in sigurantă, 
nimeni nu poate vedea soldul sau să urmarească tranzacțiile.
```

Comment:
There is a strange thing here.

Reply:
This happens when I directly copy and paste from scribbus and the text has various lines like that part, but I'm not sure why.

Reply:
> I directly copy and paste from scribbus

This is not how a migration to a different format should be done. Markdown has its own specific structure, which is not compatible with scribus'. Please make sure to export the text in the correct formatting and double check the resulting text or the format will break. See [#78 (comment)](https://github.com/monero-ecosystem/outreach-docs/pull/78/files#r248334206)